examples/SendsToken.purs

```diff
@@ -12,14 +12,17 @@ import Contract.Log (logInfo')
 import Contract.Monad (Contract, launchAff_, liftedM, runContract)
 import Contract.ScriptLookups as Lookups
 import Contract.Scripts (MintingPolicy)
-import Contract.Transaction (TransactionHash, awaitTxConfirmed)
+import Contract.Transaction
+  ( TransactionHash
+  , awaitTxConfirmed
+  , submitTxConstraintsWith
```

Comment:
Let's rename it to `submitTxFromConstraints`, because technically constraints are not submitted anywhere.

Reply:
Propose a better name if you wish, because this is also not ideal

Reply:
@klntsky I do not even understand what `submitTxConstraintsWith` supposed to mean (with what? with Lookups?), but this is name taken from Plutus.

Reply:
I think `submitTxFromConstraints` is fine. It basically does what it says. Given `Constraints` it will build balance sign and eventually submit a Tx

Reply:
> this is name taken from Plutus.

Wow. Good to know we're not the only ones bad at naming.

Anyway, let's rename and put a new entry in `doc/plutus-comparison.md`